api/camunda/version-8.6/camunda-openapi.yaml

```diff
@@ -3165,10 +3165,16 @@ components:
         - type
         - timeout
         - maxJobsToActivate
+    JobActivationResponseBase:
+      # This base schema exists to support the downstream API documentation.
+      description: Base properties for JobActivationResponse
+      type: object
     JobActivationResponse:
       description: The list of activated jobs
       deprecated: true
       type: object
+      allOf:
+        - $ref: "#/components/schemas/JobActivationResponseBase"
```

Comment:
🔧 We need to add this to the `JobActivationResult` as well. Then it will be fine.

Reply:
<img src="https://media4.giphy.com/media/AC1HrkBir3bGg/giphy.gif?cid=bd3ea57e0bqcp6kg9kk78er6tc5x91pa1f33t49cn5mzhpje&ep=v1_gifs_search&rid=giphy.gif&ct=g"/>

I take a 5 day weekend and I completely forget the last three months of work 😓